tests/phpunit/TestFramework/PhpUnit/Config/XmlConfigurationManipulatorTest.php

```diff
@@ -504,6 +508,46 @@ public function test_it_consider_as_valid_a_PHPUnit_XML_configuration_without_XS
         $this->assertTrue($this->configManipulator->validate('/path/to/phpunit.xml', $xPath));
     }
 
+    /**
+     * @dataProvider invalidSchemaProvider
+     */
+    public function test_it_cannot_validates_XML_if_schema_file_is_invalid(
+        string $xsdSchema,
+        string $errorMessage
+    ): void {
+        $xPath = $this->createXPath(<<<XML
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit
+    xsi:noNamespaceSchemaLocation="$xsdSchema"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    foo="bar"
+>
+    <invalid></invalid>
+</phpunit>
+XML
+        );
+
+        set_error_handler(
+            static function (int $type, string $message, string $file, string $line): void {
+                // Silence!
```

Comment:
This is necessary to get around the PHPUnit error handler otherwise whenever `validateSchema()` fails, it will fail there instead of continuing & collecting the errors as we expect it to do